### ISSUE TITLE
_pybgpstream.BGPStream.get_data_interfaces() appears broken

### ISSUE THREAD

**Issue by mattijsjonker** [CAIDA/bgpstream#71]
The _pybgpstream.BGPStream.get_data_interfaces() does not work for me. It leads to a segfault.

```
Python 2.7.14 |Anaconda, Inc.| (default, Dec  7 2017, 17:05:42) 
[GCC 7.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from _pybgpstream import BGPStream
>>> stream = BGPStream()
>>> stream.get_data_interfaces()
Segmentation fault
```

**Comment by digizeph** [comment]
@mattijsjonker thank you bring up this issue! To help us further investigate this issue, could you tell us the versions of BGPStream and pybgpstream that you installed in your environment? (Or the steps you followed to install the libraries)

**Comment by mattijsjonker** [comment]
Never mind, this appears to have been fixed in v1.2.0 (and I am on an older version).
